adoc/MAIN.release-notes.adoc

```diff
@@ -79,27 +130,11 @@ based on SUSE Linux Enterprise 15 SP1 and nginx in the Deployment guide:
 Updates are implemented with `skuba-update`, that makes use of the `kured`
 tool and the SLE package manager. This is implemented in the skuba-update
 tool which glues zypper and the kured tool (https://github.com/weaveworks/kured).
-Kured (KUbernetes REboot Daemon) is a Kubernetes daemonset that performs safe
+Kured (KUbernetes REboot Daemon) is a {kube} daemonset that performs safe
 automatic node reboots when the need to do so is indicated by the package
 management system of the underlying OS. Automatic updates can be manually
 disabled and configured: https://susedoc.github.io/doc-caasp/beta/caasp-admin/single-html/#_cluster_updates
 
-
-
-== Changes To Kubernetes Stack
```

Comment:
Please add back this paragraph. We were specifically asked by product management to include this if I remember correctly.